package.json

```diff
@@ -63,7 +63,7 @@
     "lodash-webpack-plugin": "^0.11.4",
     "lolex": "^1.5.2",
     "mocha": "^3.2.0",
-    "node-sass": "4.11.0",
+    "node-sass": "4.14.0",
```

Comment:
Is this required? If so, why not 4.14.1?

Reply:
@Gnafu 

![image](https://user-images.githubusercontent.com/54526151/87324553-eb047600-c538-11ea-994d-ad7e92b2ce2a.png)